app/controllers/users_controller.rb

```diff
@@ -5,6 +5,11 @@ def show
     @orders = Order.where(user_id: current_user.id)
   end
 
+  def total_price(order)
+    order.items.inject(0) { |sum, item| sum + item.price }
+  end
+  helper_method :total_price
```

Comment:
Well we could move the total_price method to the order model and then call order.total_price in the view to get the desired outcome.